language-support/scala/bindings-akka/variables.bzl

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+bindings_akka_deps = depset([
```

Comment:
As I read this everything that depends on `bindings-akka` should also depend on `bindings_akka_deps`. The `da_scala_library` rule has the [`exports`](https://github.com/bazelbuild/rules_scala/blob/master/docs/scala_library.md) attribute for that purpose.
> List of targets to add to the dependencies of those that depend on this target. Similar to the `java_library` parameter of the same name. Use this sparingly as it weakens the precision of the build graph. These must be jvm targets (scala_library, java_library, java_import, etc...)

I think putting those dependencies into `bindings-akka`'s `exports` attribute would be much easier than manually dealing with `depset`s. 

Same applies to `bindings_scala_deps`.

Reply:
yep, I need a list of transitive dependencies